src/app/actions.ts

```diff
@@ -11,6 +11,7 @@ export const handleCreate = async (token: string, payload: CreateTaskRequest) =>
       method: 'POST',
       body: JSON.stringify(payload),
     })
+    revalidateTag('getAssigneeList')
```

Comment:
We decided to remove revalidation to prevent the flickering issue and mainly real time udpates PR would not require to revalidate as revalidation philosophy has adapted to work with the supabase revalidate which resulted in high efficiency.

I think this way to re-fetch assignee each time after task is created is also increasing the overload of the app. My suggestion is, why fetch the list of assignee again if it has already been fetched. We can just mutate the existing `assignee` array with the newly selected assignee. 

The above practice is going to solve a lot of problem such as "No assignee" issue when new task is created with newly created client, and other flickering issue. Please let me know your thoughts regarding this. 

@rrojan